### PR TITLE
Add reviewed state to change note

### PR DESCRIPTION
**Pull request by leonard84**
First simple implementation of reviewed state display, for #390 and #20 .

### CODE REVIEW

**Comment by uwolfer** [comment]
@leonard84: Thanks a lot for your work! I can see, you are on the right path.

**Comment by leonard84** [comment]
@uwolfer I've tested it manually and it works as far as I can see, the only problem is that it doesn't update the review state (in the tree view) of a file if you review it in intellij. I don't know if there is an effective way to implement that, other than just refreshing the whole tree view.

In another PR, I'd like to add a filter to the panel, next to the `Diff against` box, so that you can filter by review state.

File: src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java
Comment (uwolfer) [inline src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java]: NIT: please add a space after `if`

**Review by uwolfer — COMMENTED** [review]
I've not tested this change yet, but except one minor thing, it LGTM.

**Comment by uwolfer** [comment]
> In another PR, I'd like to add a filter to the panel, next to the Diff against box, so that you can filter by review state.

Sounds good to me.

File: src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java
Comment (leonard84) [reply src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java]: Hint with the suggest (ctrl+g) functionality this can be easily applied.

```suggestion
        if (reviewed.get().contains(fileName)) {
```

**Comment by leonard84** [comment]
@uwolfer can this be merged and a new version released? The filter stuff will take longer and this already helps a lot.

File: src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java
Comment (uwolfer) [inline src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java]: You also need to add this also in `onChangeSelected` (yes, because you have forgot to add it there, it shows some flaws in this code; you might could extract these common parts).

File: src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java
Comment (uwolfer) [inline src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java:68]: This looks like a mistake in the old error. Most probably, we wanted to do `selectedChange != null`.

**Review by uwolfer — CHANGES_REQUESTED** [review]

File: src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java
Comment (uwolfer) [reply src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java]: This re-appeared (and I'm not able to apply this hint because I do not have write access to your fork). After this one is resolved, I'll merge.

**Comment by uwolfer** [comment]
@leonard84: Sorry, forgot to un-resolve one issue which reappeared. Will merge once you have fixed that.

**Comment by leonard84** [comment]
Done

The `Allow edits by maintainers` is enabled, but maybe it doesn't work with the `suggestion` feature.

**Review by uwolfer — APPROVED** [review]
Thanks for your work!

**Comment by leonard84** [comment]
@uwolfer any chance for a release?

**Comment by uwolfer** [comment]
> @uwolfer any chance for a release?

Yes, just pushed a release. Thanks for your work - really appreciated!

**Pull request merged**
